update-deps.bash

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -o pipefail
+
+declare Pkg=update-deps
+declare Version=0.1.0
+
+function msg() {
+    echo "$Pkg: $*"
+}
+
+function err() {
+    msg "$*" 1>&2
+}
+
+function main() {
+    msg "branch is ${TRAVIS_BRANCH}"
+
+    if [[ $TRAVIS_TAG =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+        msg "Updating @atomist modules to latest released versions..."
+        if ! npm install @atomist/rug@latest -S; then
```

Comment:
Is there a point in using the `-S` option if we are not committing the changes to `package.json`?